lib/power_action_utils.pm

```diff
@@ -42,7 +42,7 @@ sub prepare_system_shutdown {
         }
         console('installation')->disable_vnc_stalls;
     }
-    if (check_var('BACKEND', 'svirt')) {
+    if (check_var('VIRSH_VMM_FAMILY', 'xen')) {
```

Comment:
Update Copyright here too. I think this copyright thingy starts to be a good candidate for automation :) Probably in the make file.